pandas/tests/extension/base/methods.py

```diff
@@ -438,6 +438,18 @@ def test_searchsorted(self, data_for_sorting, as_series):
         sorter = np.array([1, 2, 0])
         assert data_for_sorting.searchsorted(a, sorter=sorter) == 0
 
+        # arr containing na value
```

Comment:
I think since this is applicable to pandas specific masked arrays + arrow array, this test should live in those specific test files. Technically these base extension tests should be passable for any EA developer who might not define `searchsorted` to error in this way